public/app/plugins/datasource/loki/datasource.test.ts

```diff
@@ -133,19 +134,19 @@ describe('LokiDatasource', () => {
       dsMaxLines: number | undefined,
       expectedMaxLines: number
     ) => {
-      let settings: any = {
+      let settings = {
```

Comment:
Would it be possible to use type definition (`let settings: DataSourceInstanceSettings = ...`) instead of type assertion in some of these fixes?

Reply:
But I re-visited changes and actually found 2 places where `as...` could be removed. Done in https://github.com/grafana/grafana/pull/53114/commits/3e8c0a9d496cea98699cf5b3b04a87c19ec543c4